package.json

```
@@ -65,7 +67,7 @@
     "ember-export-application-global": "^2.0.1",
     "ember-load-initializers": "^2.1.1",
     "ember-maybe-import-regenerator": "^0.1.6",
-    "ember-qunit": "^4.6.0",
+    "ember-qunit": "5.0.0-beta.2",
```

Should probably be `^` 

```suggestion
    "ember-qunit": "^5.0.0-beta.2",
```